.github/workflows/lint.yml

```diff
@@ -19,5 +19,5 @@ jobs:
         version: 16.x
     - name: Lint
       run: |
-        npm ci
-        npm run lint
+        yarn
```

Comment:
We use npm in the repo so i'd prefer we were consistent - why did we switch to yarn here?

Reply:
In the process of doing this basically to every repo in the electron org I think this was the only one that wasn't on yarn and I couldn't make npm update all the nested dependencies without just blowing away the lockfiles completely. Seemed easier and more consistent to be like all the other repos we have with yarn lockfiles

By comparison all the other ones I did with yarn were one or two targetted commands that resulted in very small diffs.

E.g. the commit you linked further down is a multi thousand line diff for a single dependency change.  That's unreadable and unreviewable.